model_specific/SM/standard_model.hpp

```diff
@@ -657,6 +654,7 @@ class Standard_model : public Beta_function {
    Eigen::Array<double,3,1> MFu{Eigen::Array<double,3,1>::Zero()};
    Eigen::Array<double,3,1> MFe{Eigen::Array<double,3,1>::Zero()};
    double MVWp{};
```

Comment:
Please add a comment: `// running W boson mass`

Reply:
Done in 673aff15e9eb625a99aabbc1f0092c29ab66a93e.